### PR TITLE
feat: add storage group page

### PR DESCRIPTION
**Pull request by astandrik**
[Stand](http://ydb-vla-dev02-005.search.yandex.net:8765/monitoring/cluster/storage)

Closes https://github.com/ydb-platform/ydb-embedded-ui/issues/1159

### Test Status: <span style="color: orange;">⚠️ FLAKY</span>
📊 [Full Report](https://ydb-platform.github.io/ydb-embedded-ui/1289/)

| Total | Passed | Failed | Flaky | Skipped |
|:-----:|:------:|:------:|:-----:|:-------:|
| 124 | 116 | 0 | 8 | 0 |

### Bundle Size: 🔺
Current: 78.96 MB | Main: 78.84 MB
Diff: +0.12 MB (0.15%)

⚠️ Bundle size increased. Please review.

<details>
<summary>ℹ️ CI Information</summary>

- Test recordings for failed tests are available in the full report.
- Bundle size is measured for the entire 'dist' directory.
- 📊 indicates links to detailed reports.
- 🔺 indicates increase, 🔽 decrease, and ✅ no change in bundle size.
</details>

### CODE REVIEW

File: src/containers/StorageGroupPage/StorageGroupPage.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    &__tabs {
+        @include tabs-wrapper-styles();
+    }
+
+    &__group-disk {
+        width: 150px;
+    }
+}
```
Comment (artemmufazalov) [inline src/containers/StorageGroupPage/StorageGroupPage.scss:45]: You don't have `controls`, `group-disks` and `group-disk` on this page

File: src/components/StorageGroupInfo/i18n/en.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "used-space": "Used Space",
+  "usage": "Usage",
+  "read-throughput": "Read Throughput",
+  "write-throughput": "Write Throughput",
+  "links": "Links",
+  "storage-group-page": "Storage Group Page",
+  "developer-ui": "Developer UI",
+  "yes": "Yes",
+  "no": "No",
+  "storage-group-title": "Storage Group",
```
Comment (artemmufazalov) [inline src/components/StorageGroupInfo/i18n/en.json:18]: Redundant field

File: src/components/StorageGroupInfo/i18n/en.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  "kind": "Kind",
+  "degraded": "Degraded",
+  "change-time": "Change Time",
+  "group-generation": "Group Generation",
+  "latency": "Latency",
+  "acquired-units": "Units",
+  "allocation-units": "Units",
```
Comment (artemmufazalov) [inline src/components/StorageGroupInfo/i18n/en.json:25]: Why do you need both?

**Review by artemmufazalov — CHANGES_REQUESTED** [review]

**Comment by Raubzeug** [comment]
A bug when scrolling:

Screenshot 2024-09-17 at 09 59 08

File: src/components/StorageGroupInfo/StorageGroupInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    if (valueIsDefined(Latency)) {
+        storageGroupInfoSecondColumn.push({
+            label: storageGroupInfoKeyset('latency'),
+            value: <EntityStatus status={Latency} />,
+        });
+    }
+    if (valueIsDefined(LatencyPutTabletLog)) {
+        storageGroupInfoSecondColumn.push({
+            label: storageGroupInfoKeyset('latency-put-tablet-log'),
+            value: `${LatencyPutTabletLog} ms`,
```
Comment (Raubzeug) [inline src/components/StorageGroupInfo/StorageGroupInfo.tsx:130]: Here and farther lets add spaces between the digits of the number (like it is done in `Available Space` field)

File: src/components/StorageGroupInfo/StorageGroupInfo.tsx [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+    }
+    if (valueIsDefined(Write)) {
+        storageGroupInfoSecondColumn.push({
+            label: storageGroupInfoKeyset('write-throughput'),
+            value: bytesToSpeed(Number(Write)),
+        });
+    }
+
+    return (
+        <div className={b('wrapper', className)}>
```
Comment (Raubzeug) [inline src/components/StorageGroupInfo/StorageGroupInfo.tsx:165]: lets use `Flex` component

File: src/components/StorageGroupInfo/StorageGroupInfo.tsx [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+            <div className={b('col')}>
```
Comment (Raubzeug) [inline src/components/StorageGroupInfo/StorageGroupInfo.tsx:166]: do we need this wrapper?

File: src/containers/StorageGroupPage/StorageGroupPage.tsx [hunk 7]
```diff
@@ -0,0 +1,201 @@
...
+
+    const renderError = () => {
+        if (!groupQuery.error && !nodesQuery.error) {
+            return null;
+        }
+        return <ResponseError error={groupQuery.error || nodesQuery.error} />;
+    };
+
+    return (
+        <div className={storageGroupPageCn()}>
```
Comment (Raubzeug) [inline src/containers/StorageGroupPage/StorageGroupPage.tsx:191]: `Flex`?

**Review by Raubzeug — CHANGES_REQUESTED** [review]

**Review by Raubzeug — APPROVED** [review]

**Review by artemmufazalov — APPROVED** [review]

**Pull request merged**